cnv/cnv_users_guide/cnv-enabling-user-permissions-to-clone-datavolumes.adoc

```diff
@@ -0,0 +1,24 @@
+[id="cnv-enabling-user-permissions-to-clone-datavolumes"]
+= Enabling user permissions to clone DataVolumes across namespaces
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-enabling-user-permissions-to-clone-datavolumes
+toc::[]
+
+The isolating nature of namespaces means that users cannot by default 
+clone resources between namespaces. 
+
+To enable a user to clone a virtual machine to another namespace, a 
+user with the ClusterAdmin role must create a new ClusterRole. Bind 
+this ClusterRole to a user to enable them to clone virtual machines
+to the destination namespace. 
+
+.Prerequisites
+
+* Only a user with the xref:../../authentication/using-rbac.adoc#default-roles_using-rbac[`cluster-admin`]
+role can create ClusterRoles.
+
+include::modules/cnv-about-datavolumes.adoc[leveloffset=+1]
```

Comment:
(If you include `cnv-about-datavolumes.adoc`, make sure you add this assembly to the "Module included in the following assemblies" list in that file.)

Reply:
Good catch!